granary/twitter.py

```diff
@@ -291,6 +297,29 @@ def _user():
           retweet_calls += 1
           cache_updates['ATR ' + id] = count
 
+    if fetch_quotes:
+      # organize quote tweets by id of the original
+      quote_tweets = {}
+      # search for 5 ids at a time based on the guideline
+      # "Limit your searches to 10 keywords and operators."
+      # https://dev.twitter.com/rest/public/search
+      for five_tweets in [tweets[i:i + 5] for i in xrange(0, len(tweets), 5)]:
+        url = API_SEARCH_URL % {
+          'q': urllib.quote_plus(
+            ' OR '.join((t['id_str'] for t in five_tweets))),
```

Comment:
funny, reminds me of https://github.com/snarfed/bridgy/blob/444b05710c15bdd2713e5cb8fe2f15a478d0e8cd/models.py#L273 and https://github.com/snarfed/bridgy/blob/444b05710c15bdd2713e5cb8fe2f15a478d0e8cd/twitter.py#L67 etc